src/components/Home/HomeMap.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import { Menu, Popup, Icon, Label } from 'semantic-ui-react';
+import GoogleMapReact from 'google-map-react';
+
+const style = {
+  borderRadius: 0,
+  border: '1px solid white',
+  opacity: 0.7,
+  backgroundColor: 'grey',
+  color: 'white'
+};
```

Comment:
Try not to do inline-styling. Otherwise, when you have to make future styling changes, it won't be overwritten because inline-styles take the most precedence behind `!important`.